pages/vi/faq.md

```diff
@@ -13,6 +13,7 @@
 - **Q9:** [How long do I have to complete these steps?](#Q9:_How_long_do_I_have_to_complete_these_steps?)
 
 ## Technical Questions
+- **Q1:** [What if you encounter *Host key verification fail* or *Add correct host key in C:\\Users\\USER/.ssh/known_hosts to get rid of this message*?](#Q1:_What_if_you_encounter_Host_key_verification_fail?)
```

Comment:
Q11?

Reply:
I was intending to add it in technical question.

Reply:
yes
we wanna also keep them unique by number ;)
so that there is no misunderstanding